code/style.css

```diff
@@ -1,5 +1,360 @@
+  padding: 8px 10px;
+  text-decoration: none;
+  border-radius: 5px;
+}
 
 body {
-  background-color: peachpuff;
+  margin: 00px;
+  color: black;
+  box-sizing: border-box;
+}
+
+
+body h2 {
+  font-weight: 100;
+  font-size: 30px;
+  padding: 20px;
+  margin: 0;
+  display: flex;
+  justify-content: center;
+}
+
+a.button {
+  background-color: #2D519F;
+  border-radius: 20px;
+  color: #D9DEE4;
+  font-weight: 500;
+  padding: 10px 30px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 15px;
+  margin: 10px 0px;
+  max-width: 200px;
+  align-self: center;
+
+}
+
+.welcome-container {
+  background-color: #2D519F;
+  padding: 50px 50px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.welcome-container .text-container {
+  background-color: white;
+  height: 100%;
+  padding: 30px;
+  margin-bottom: 30px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+
+.shop-container {
+  background-color: #0B2D78;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 30px;
+  align-items: center;
+}
+
+.shop-container .shop-cart {
+  background-color: #fff;
+  height: 100%;
+  min-width: none;
+  max-width: 400px;
+  border-radius: 20px;
+  margin: 0px 50px 50px 50px;
+  padding: 0;
+  box-shadow: 0px 12px 10px 0 #0000008f;
+  text-decoration: none;
+  display: flex;
+  flex-flow: row wrap;
+}
+
+.shop-container img {
+  width: 100%;
+  max-width: 400px;
+  border-radius: 12px 12px 0 0;
+  margin: 0;
 }
+
+.shop-container p {
+  padding: 20px;
+}
+
+.shop-container .text-container {
+  background-color: white;
+  color: black;
+  height: 100%;
+  padding: 30px;
+  margin-bottom: 30px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+}
+
+
+.news-container {
+  background-color: #753A90;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  color: white;
+}
+
+.news-container div {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+  padding: 30px;
+  box-sizing: border-box;
+  margin: auto;
+}
+
+.news-container .news-text {
+  background-color: #0B2D78;
+  color: white;
+  height: 100%;
+  padding: 20px 20px;
+  margin: 50px 0px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.news-container p {
+  color: white;
+}
+
+.news-container .news-text .button {
+  background-color: white;
+  color: black;
+}
+
+.news-container img {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  max-width: 300px;
+  padding: 20px;
+  border-radius: 50%;
+
+}
+
+.portfolio-container {
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.gallery {
+  padding: 0px 50px;
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.photo {
+  padding: 10px 10px 10px 10px;
```

Comment:
If you want the same amount for all directions, as 10px here, you could simply write `padding: 10px`, as it will apply the same amount to all sides. If you want to add different to top and bottom, you could write `padding: 20px 10px`, and it would get 20px on top and bottom, and 10px on left and right. 🌈